package.json

```diff
@@ -30,20 +31,21 @@
         "install:all-android": "cd components && yarn install && cd .. && yarn install:showcase-android && yarn install:storybook-android && yarn install:storybook-api",
         "link:components": "bash ./scripts/linkComponents.sh",
         "start": "yarn start:showcase",
-        "start:showcase": "yarn start:showcase-ios",
-        "start:showcase-ios": "yarn install:showcase-ios && cd demos/showcase && yarn ios",
-        "start:showcase-android": "yarn install:showcase-android && cd demos/showcase && yarn android",
+        "start:showcase": "yarn initialize && yarn start:showcase-ios",
```

Comment:
by running `yarn start:showcase`, the user is initializing the submodule twice

Reply:
It shouldn't hurt (the second time it just runs prints the output saying it's linked without doing any actions), but I moved the initialization to the install script to be cleaner, so it should only happen once now.